airflow/providers/amazon/aws/operators/s3.py

```diff
@@ -683,7 +683,12 @@ def execute(self, context: Context):
             self.delimiter,
         )
 
-        return hook.list_keys(bucket_name=self.bucket, prefix=self.prefix, delimiter=self.delimiter)
+        return hook.list_keys(
+            bucket_name=self.bucket,
+            prefix=self.prefix,
+            delimiter=self.delimiter,
+            apply_wildcard=True,
```

Comment:
This will change the behaviour of this operator with no way to disable it. Can you add a new field to the operator itself for applying wildcards and set the default to False.

Reply:
Thanks for the suggestion. Fixed.